UnityProject/Assets/Scripts/Objects/Research/ResearchPointMachine.cs

```diff
@@ -91,14 +91,9 @@ public void PowerNetworkUpdate(float voltage)
 
 		public virtual void StateUpdate(PowerState state)
 		{
-			if (state == PowerState.Off)
-			{
-				//Machine loses connection to server on power loss
-				UnSubscribeFromServerEvent();
```

Comment:
Why is this removed?

Reply:
I personally think it does not add much and creates an issue.
Power on the station flickers off, even for a second quite frequently (often if engi turns off the plasma gen to run the stations reactor or similliar). Having the science department reconnect every machine in their department every time there is a power flicker seems awfully tedious and not adding much gameplay.
Also, this often triggers before power is fully initalised, meaning at round start, all resourcepoint machines disconnect from the server, meaning you have to go and reconnect them all everytime. For these reasons I just removed it

Reply:
make it time dependent. a flicker won't hurt, but power goes out for 10 minutes and you have to set things up again

Reply:
the APC powerable component doesnt have support for that right now, If we wanted to do that, that can be added at a later state. This just removes the logic from the APCpowerable component as the functionality that results in is not ideal

Reply:
okay that's fair enough